linux_os/guide/system/software/integrity/endpoint_security_software/mcafee_security_software/mcafee_hbss_software/package_MFEhiplsm_installed/rule.yml

```diff
@@ -24,23 +22,40 @@ references:
     nist: CM-6(a)
     nist-csf: DE.AE-1,DE.AE-2,DE.AE-3,DE.AE-4,DE.CM-1,DE.CM-5,DE.CM-6,DE.CM-7,DE.DP-2,DE.DP-3,DE.DP-4,DE.DP-5,ID.RA-1,PR.AC-5,PR.DS-5,PR.IP-8,PR.PT-4,RS.AN-1,RS.CO-3
     pcidss: Req-11.4
-    srg: STG-OS-000480-GPOS-00227
+    srg: SRG-OS-000480-GPOS-00227,SRG-OS-000196
     isa-62443-2013: 'SR 2.10,SR 2.11,SR 2.12,SR 2.4,SR 2.8,SR 2.9,SR 3.1,SR 3.3,SR 3.5,SR 3.8,SR 3.9,SR 4.1,SR 4.3,SR 5.1,SR 5.2,SR 5.3,SR 6.1,SR 6.2,SR 7.1,SR 7.6'
     isa-62443-2009: 4.2.3,4.2.3.12,4.2.3.7,4.2.3.9,4.3.3.4,4.3.4.5.2,4.3.4.5.6,4.3.4.5.7,4.3.4.5.8,4.3.4.5.9,4.4.3.2,4.4.3.3,4.4.3.4
     cobit5: APO01.06,APO07.06,APO08.04,APO10.05,APO11.06,APO12.01,APO12.02,APO12.03,APO12.04,APO12.06,APO13.01,APO13.02,BAI08.02,BAI08.04,DSS01.03,DSS01.05,DSS02.04,DSS02.05,DSS02.07,DSS03.01,DSS03.04,DSS03.05,DSS04.05,DSS05.01,DSS05.02,DSS05.04,DSS05.05,DSS05.07,DSS06.01,DSS06.02,MEA03.03,MEA03.04
     iso27001-2013: 'A.10.1.1,A.11.1.4,A.11.1.5,A.11.2.1,A.12.1.1,A.12.1.2,A.12.4.1,A.12.4.3,A.12.5.1,A.12.6.1,A.12.6.2,A.13.1.1,A.13.1.2,A.13.1.3,A.13.2.1,A.13.2.3,A.13.2.4,A.14.1.2,A.14.1.3,A.14.2.7,A.14.2.8,A.15.2.1,A.16.1.1,A.16.1.2,A.16.1.3,A.16.1.4,A.16.1.5,A.16.1.6,A.16.1.7,A.18.1.4,A.18.2.2,A.18.2.3,A.6.1.2,A.7.1.1,A.7.1.2,A.7.3.1,A.8.2.2,A.8.2.3,A.9.1.1,A.9.1.2,A.9.2.3,A.9.4.1,A.9.4.4,A.9.4.5,Clause 16.1.2,Clause 7.4'
     cis-csc: 1,11,12,13,14,15,16,18,19,2,3,4,5,6,7,8,9
+    stigid@rhel7: RHEL-07-020019
 
 ocil_clause: 'the HBSS HIPS module is not installed'
 
 ocil: |-
     To verify that McAfee HIPS is installed, run the following command(s):
     <pre>$ rpm -q MFEhiplsm</pre>
 
+{{% if product in ["rhel7", "ol7", "rhel8", "ol8"] %}}
+conflicts:
+    - selinux_state
+{{% endif %}}
```

Comment:
We don't need these jinja statements. Jinja shouldn't revolve around yaml keys. It makes it hard for non-jinja and python consumption which is what we are going to need to do in the future for integration into other standards.

Reply:
Do you have any other suggestions how to make this conditional? Currently we can't make this working without conditionals, because some products are lacking the selinux_enabled rule.

Reply:
I would suggest configuring the prodtype to be set correctly instead of removing it.

Reply:
@vojtapolasek I agree with Gabe that this rule should have a prodtype set, as "package installed/removed" rules work only with products where this package exists under this precise name.
@redhatrises In this case, the conditional shouldn't be needed, as the build system should be able to deduce that if this rule conflicts with a rule that isn't in the benchmark, there is nothing to do. But the flexibility Jinja provides to content authors is a great asset. When mentioning jinja-less rule yamls, then check out `build/<product>/rules/` - isn't that good enough?